src/app/shared/issue-tables/search-filter.ts

```diff
@@ -51,6 +56,11 @@ function matchesDuplicatedIssue(issueService: IssueService, id: number, searchKe
   return duplicatedIssuesContainsSearchKey(duplicatedIssues, searchKey);
 }
 
+function matchesTitle(issue: Issue, searchKey: string): boolean {
+  const searchStr = (issue.title + ' #' + issue.id).toLowerCase();
```

Comment:
This approach is fine for now but this might be hard for future devs to find this should there be a change in the formatting in the tables.

Reply:
Remember to request for review if this PR is complete